### ISSUE TITLE
RANSAC

### ISSUE THREAD

**Issue by vadixidav** [rust-cv/cv#2]
The [`sample-consensus`](https://github.com/rust-cv/sample-consensus) crate provides abstractions for consensus algorithms. Currently, `arrsac` is provided as a state-of-the-art consensus algorithm. However, it is useful to be able to compare the original RANSAC algorithm, which is commonly used as a benchmark or when comparing algorithms that aren't sample consensus algorithms when used in conjunction with a sample consensus algorithm.

**Comment by higumachan** [comment]
@vadixidav 

I'd like to implement this one, how about you?

**Comment by vadixidav** [comment]
@higumachan Absolutely. You will want to fork this repository: https://github.com/rust-cv/ransac. It os currently empty. I would refer to the ARRSAC repository as an example of implementation. RANSAC should be simpler than ARRSAC, so this should be a great first issue.

**Comment by vadixidav** [comment]
You can find the ARRSAC repo here: https://github.com/rust-cv/arrsac
